src/App.tsx

```diff
@@ -17,18 +77,39 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filter={filter}
+                query={query}
+                onFilterChange={handleFilterChange}
+                onSearchChange={handleSearchChange}
+                onClearSearch={handleClearSearch}
+              />
```

Comment:
Ensure that all form elements and buttons have accessible text or `aria-labels` to describe their actions for screen readers. This is important for users relying on assistive technologies.